include/nuttx/serial/tioctl.h

```diff
@@ -202,6 +202,14 @@
 
 #define TIOCSLINID      _TIOC(0x0037) /* Master send one LIN header with specified LIN identifier: uint8_t */
 
+/* Single-wire UART Duplex */
+
+#define TIOCSSINGLEWIREDUPLEX _TIOC(0x0038)  /* Set single-wire duplex mode */
```

Comment:
I fund the name confusing, it is the opposite of what it is. It is SIMPLEX, but, I would call it TIOCSSINGLEWIREUNIDIRECTIONAL.